game.py

```diff
@@ -30,29 +30,49 @@ class AppleFinder(GridGame):
     NUM_OF_PITS_PER_LEVEL = 8
     MAX_TURNS = 300
 
-    PLAYER = '@'
+    # will start from the base tile set
+
+    PLAYER = chr(255)
+    WATER = chr(247)
+    MOUNTAIN = chr(30)
+    GROUND = chr(176)
+    HOLE = ' '
+    BASE = '#'
+    # will start by using numbers for plant growth stages
+    ROCK = chr(239)
+    BEAST = chr(2)
+    TREE1 = chr(5)
+    TREE2 = chr(6)
+    EMPTY = GROUND
+    PIT = HOLE
+    APPLE = chr(15)
     APPLE = 'O'
-    EMPTY = ' '
-    PIT = '^'
+    BASE = chr(233)
 
     def __init__(self, random):
         self.random = random
         self.running = True
         self.in_pit = False
-        centerx = self.MAP_WIDTH // 2
-        centery = self.MAP_HEIGHT // 2
-        self.player_pos = [centerx, centery]
+
+
+        # place robot and base randomly
+        x = self.random.randint(0, self.MAP_WIDTH - 1)
+        y = self.random.randint(0, self.MAP_HEIGHT - 1)
+        self.base_bos = [x, y]
```

Comment:
Did you mean `base_pos`? If not I am not sure what `bos` is maybe add a comment.

Reply:
self.base_bos is a typo that is never used... can I edit it in this pull request?

Reply:
I believe so but I am not sure

Reply:
I deleted this line. I realize what I was going to use it for (the position of the base), but that is just also the initial placement location, and the base can't move, so I did not need "base_pos" (which I typoed as base_bos). I removed this from the committed file.